packages/example-app-base/src/components/ApiReferencesTable/ApiReferencesTable.tsx

```diff
@@ -0,0 +1,512 @@
+        />
+      </Stack>
+    );
+  }
+
+  private _renderExtends(): JSX.Element | undefined {
+    const { extendsTokens } = this.props;
+
+    return this._parseILinkTokens(true, extendsTokens);
+  }
+
+  private _renderDescription(): JSX.Element | undefined {
+    const { description } = this.props;
+
+    return description ? <Text variant={'medium'}>{description}</Text> : undefined;
+  }
+
+  private _renderTitle(): JSX.Element | undefined {
+    const { title, name } = this.props;
+
+    return title ? (
+      <Text variant={'xLarge'} id={name}>
+        {title}
+      </Text>
+    ) : (
+      undefined
+    );
+  }
+
+  private _onRenderRow = (props: IDetailsRowProps, defaultRender?: IRenderFunction<IDetailsRowProps>): JSX.Element => {
+    const { item } = props;
+    const rowStyles: Partial<IDetailsRowStyles> = {
+      root: {
+        selectors: {
+          ':hover': {
+            background: 'none'
+          }
+        }
+      },
+      isMultiline: {
+        wordBreak: 'break-word'
+      }
+    };
+
+    if (item.deprecated === true) {
+      const deprecatedStyles: Partial<IDetailsRowStyles> = {
+        root: {
+          background: DEPRECATED_ROW_COLOR,
+          selectors: {
+            ':hover': {
+              background: DEPRECATED_ROW_COLOR
+            }
+          }
+        }
+      };
+
+      return <DetailsRow {...props} styles={{ ...rowStyles, ...deprecatedStyles }} />;
+    }
+
+    return <DetailsRow {...props} styles={rowStyles} />;
+  };
+
+  private _parseILinkTokens(extend: boolean, linkTokens?: ILinkToken[]): JSX.Element | undefined {
+    if (linkTokens && linkTokens.length > 0) {
+      if (extend) {
+        return (
+          <Text variant={'small'}>
+            {'Extends '}
+            {this._parseILinkTokensHelper(linkTokens, true)}
+          </Text>
+        );
+      } else {
+        return (
+          <Text variant={'small'}>
+            <code>{this._parseILinkTokensHelper(linkTokens, false)}</code>
+          </Text>
+        );
+      }
+    }
+
+    return undefined;
+  }
+
+  private _parseILinkTokensHelper = (linkTokens: ILinkToken[], extend: boolean): JSX.Element | undefined => {
+    return (
+      <>
+        {linkTokens.map((token: ILinkToken, index: number) => {
+          let match: RegExpMatchArray | null = null;
+          let hash: string = '/components/';
+          // get hash to set correct href value on the links for the local site vs. the Fabric site
+          if (this._baseUrl !== '') {
+            match = this._baseUrl.match(hashRegex);
+            if (match !== null) {
+              hash = match[0];
+            }
+          }
+          if (token.pageKind && token.hyperlinkedPage) {
+            let href;
+            const split = this._baseUrl.split('#');
+            const cleanedSplit = split.filter(value => !!value);
+
+            // whether the link should be opened in a new tab, defaults to true
+            let newTab = true;
```

Comment:
We should look at similar doc sites to see if their links open in a new tab

Reply:
This was design discussed with @aditima @Vitalius1 @KevinTCoughlin @aftab-hassan @margoflynn and @iclanton. If we're not a fan of this, we can iterate off of it, but we thought this would be the best approach for now.

Reply:
The justification for the "New tab" experience when clicking a hyper-link was so that folks reading API references did not lose their place (scroll position and URL) when looking up related API references. Let's see how it works in practice and revise if need be.

Reply:
I can kind of see that perspective, but personally I find it incredibly annoying if I *didn't* want a new tab, because then I have to go back and close the old one--whereas if I *do* want a new tab, I can easily make that happen with ctrl/cmd+click. For scroll position, natively I think browsers tend to do a pretty good job remembering it these days, but that might be negated by our custom scroll handling.

Also, FWIW, the handful of the doc sites I looked at just now **don't** use new tabs: [Node docs](https://nodejs.org/api/crypto.html), [MDN](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference), [.NET docs](https://docs.microsoft.com/en-us/dotnet/) (and other stuff on docs.microsoft.com), [Java docs](https://docs.oracle.com/javase/10/docs/api/overview-summary.html).

Reply:
I agree with @ecraig12345 here. Most ideal is some kind of "peek" but navigating to the linked item is far more aligned with what I usually encounter, including in various IDEs (they don't open the code in a new tab when you navigate to definition).

Reply:
IIRC our custom scroll behavior was the problem. You can see this today on our documentation site where you are reset to `top: 0`. We felt that given the value of the information we are now surfacing, that the trade-off was acceptable in the short-term.

@natalieethell, let's follow-up and see if we can fix the root cause. Then we can relax enforcing new tab behavior, which I think we're all in favor of if scroll position is restored.